backend/queries/queries.go

```diff
@@ -5,100 +5,140 @@ import (
 )
 
 type MegaJSONStruct struct {
-	RepoContrib     repositoriesContributedTo
-	Pr              pullRequests
-	IssCreated      issuesCreated
-	PRContributions linesofCodeInPRs
-	PRCommits       commitsOnPRs
-	AccountInfo     accountInformation
+	AccountInfo          accountInformation
+	PRs                  PullRequests
+	MergedVsNonMergedPRs PRsMergedVsNot
+	InvolveIssues        InvolvedIssues
+	OpenVsClosedIssues   IssuesOpenVsClosed
+	ReposContribedTo     ReposContributedTo
+	PodInfo              PodInformation
 }
 
-type linesofCodeInPRs struct {
+type accountInformation struct {
+	User struct {
+		Name            graphql.String
+		AvatarUrl       graphql.String
+		Bio             graphql.String
+		Company         graphql.String
+		Location        graphql.String
+		Url             graphql.String
+		WebsiteUrl      graphql.String
+		TwitterUsername graphql.String
+	} `graphql:"user(login: $username)"`
+}
+
+type PullRequests struct {
 	User struct {
 		PullRequests struct {
 			TotalCount graphql.Int
 			Nodes      []struct {
 				Url         graphql.String
+				Title       graphql.String
 				CreatedAt   graphql.String
+				MergedAt    graphql.String
 				MergeCommit struct {
 					Additions graphql.Int
 					Deletions graphql.Int
 				}
+				Commits struct {
+					TotalCount graphql.Int
+				} `graphql:"commits(first: 1)"`
+				Repository struct {
+					Url             graphql.String
+					Name            graphql.String
+					PrimaryLanguage struct {
+						Name graphql.String
+					}
+				}
 			}
-		} `graphql:"pullRequests(first: 50, states:MERGED)"`
-	} `graphql:"user(login:$username)"`
+		} `graphql:"pullRequests(first:40, states: MERGED)"`
+	} `graphql:"user(login: $username)"`
 }
 
-type commitsOnPRs struct {
+type InvolvedIssues struct {
 	User struct {
-		PullRequests struct {
+		IssueComments struct {
 			TotalCount graphql.Int
 			Nodes      []struct {
-				Url       graphql.String
-				CreatedAt graphql.String
-				Commit    struct {
-					TotalCount graphql.Int
-				} `graphql:"commits(first: 1)"`
+				Issue struct {
+					Title graphql.String
+				}
+				Url        graphql.String
+				Repository struct {
+					Name            graphql.String
+					Url             graphql.String
+					PrimaryLanguage struct {
+						Name graphql.String
+					}
+				}
 			}
-		} `graphql:"pullRequests(first: 30, states:MERGED)"`
-	} `graphql:"user(login:$username)"`
+		} `graphql:"issueComments(first: 40)"`
+	} `graphql:"user(login: $username)"`
+}
+
+type IssuesOpenVsClosed struct {
+	User struct {
+		Issues struct {
+			TotalCount graphql.Int
+			Nodes      []struct {
+				State graphql.String
+			}
+		} `graphql:"issues(first: 30, states:[OPEN,CLOSED])"`
+	} `graphql:"user(login: $username)"`
```

Comment:
I wonder if we could make this into a simpler query like
```graphql
User {
   Issues {
      totalcount
   } states:[OPEN]
   Issues {
      totalcount
   } states:[CLOSED]
} 
```

But I'm not sure whether that would be at all possible

Reply:
The query in graphQL would be:
```
{
  user(login: "IamCathal") {
    open: issues(first: 20, states:[OPEN]) {
      issueCount: totalCount
    }
    closed: issues(first: 20, states:[CLOSED]) {
      issueCount: totalCount
    }
  }
}
``` 

Which gives:
```json
{
  "data": {
    "user": {
      "open": {
        "issueCount": 0
      },
      "closed": {
        "issueCount": 11
      }
    }
  }
}
```

Notice in the graphQL query it has `closed: issues` and in the output those keys are open and closed. We can't use those aliases in the golang library we use so the existing query is the simplest solution